### PR TITLE
Easi 2686/user table migration cleanup

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# EASI-2686

## Changes and Description

- Remove old baseStruct and baseTaskList
    - Rename baseStructUserTable to baseStruct
    - Rename baseTaskListUserTable to baseTaskList
- Removed redundant created by and modified by account fetching logic
   - implemented embedded createdByUserAccount and modifiedByUserAccount in BaseStruct and ExistingModel

- Remove IT tools from front end and backend

- Remove python migration script

- Remove unused EUA generation tools

## How to test this change
1. Make sure unit tests pass
2. Make sure that you can run the app
3. Look through the code for any other instances of old EUAIDs that can be removed from the app.

## PR Author Review Checklist

- [x] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/models/base_struct.go [hunk 1]
```diff
@@ -19,42 +16,49 @@ type IBaseStruct interface {
...
+	return nil
 }
 
 // GetID returns the ID property for a PlanBasics struct
 func (b baseStruct) GetID() uuid.UUID {
 	return b.ID
 }
 
-// GetModifiedBy returns the ModifiedBy property for a PlanBasics struct
+// GetModifiedBy returns the ModifiedBy property for an i base struct
```
Comment (ClayBenson94) [inline pkg/models/base_struct.go:45]:
```suggestion
// GetModifiedBy returns the ModifiedBy property for an IBaseStruct
```

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]
I think there's also some Cypress tests related to IT Tools that could probably go as well

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]
Found a few other references to IT Tools that can be nixed:

## Backend
https://github.com/CMSgov/mint-app/blob/aae40dbc7ff7d5370863e71713703e726a4cc06e/pkg/constants/constants.go#L23
https://github.com/CMSgov/mint-app/blob/024b85bad0a37d82f3e103aefb6e9ea3f986e9fc/pkg/graph/schema.graphql#L1545
https://github.com/CMSgov/mint-app/blob/3422a0933659fd88d7572d94492dc47def8b2af6/pkg/models/shared_enums.go#L69
https://github.com/CMSgov/mint-app/blob/60a518113d8d0bf655b37ff3f2df575a556a5428/pkg/storage/SQL/plan_it_tools/create.sql#L1

## Frontend
(@patrickseguraoddball - can you take a look and see if there's any others I missed?)
[This whole folder](https://github.com/CMSgov/mint-app/blob/7ba2329e4259dd5971a221b582cc70036a3f764b/src/components/ITToolsSummary)
https://github.com/CMSgov/mint-app/blob/3612bbb6efb9c19a0fb35ea276b042b6827bcb39/src/utils/export/CsvData.ts#L155
https://github.com/CMSgov/mint-app/blob/ce3ec206ed9640b8e5f526038b38fe0c4a7fce36/src/views/ModelPlan/ReadOnly/_components/MobileNav/index.tsx#L67
https://github.com/CMSgov/mint-app/blob/f856514def7b56b9336b3b5daeb2e417cdbaf658/src/components/ITToolsWarning/index.tsx (Does this make sense to remove?)

File: pkg/models/base_struct.go (hunk 1 above)
Comment (StevenWadeOddball) [reply pkg/models/base_struct.go]: I implemented this, thanks!

**Comment by StevenWadeOddball** [comment]
Thanks @ClayBenson94! I implemented the other changes you noted. I left https://github.com/CMSgov/mint-app/blob/f856514def7b56b9336b3b5daeb2e417cdbaf658/src/components/ITToolsWarning/index.tsx because I wasn't sure if it is also being used for the IT LEAD / IT Solutions section as well.

**Review by ClayBenson94 — APPROVED** [review]

**Pull request merged**